api/rs/slint/Cargo.toml

```diff
@@ -213,5 +213,6 @@ rustdoc-args = [
   # "docs/src/utils/slint-docs-preview.html",
   "--html-in-header",
   "docs/src/utils/slint-docs-highlight.html",
+  "--cfg", "docsrs"
```

Comment:
I don't think this is necessary. I believe docsrs already do it

Reply:
Good catch. I took this from wgpu, but it seems it's indeed "standard" now.